### ISSUE TITLE
Catch (mass) re-numbering

### ISSUE THREAD

**Issue by dch0ph** [gregrs-uk/fhodot#26]
South Tyneside seems to have a had a mass update of its FHRS IDs, and so the match rate has gone from overnight from ~30% to 0%, which is very disheartening. 

Perhaps this could be managed? It would require comparing looking specifically at "old" and "new" FHRS records and identifying a changed FHRS ID where all the address data remained the same. The "suggested match" layer could then be extended to propose "New FHRS ID?".

**Comment by jnicho02** [comment]
You're quite right. It has happened in a number of other Local Authorities because someone is hawking a new system around. When they transfer data they regenerate all the identifiers :(

**Comment by dch0ph** [comment]
>  I've started [fixing OSM](https://www.openstreetmap.org/changeset/148318717), we'll see how much more needs to be done after the website next updates.

That's very heroic - I was looking for a lazy solution!

**Comment by gregrs-uk** [comment]
As @Cj-Malone said the postcode matches table is the quickest way currently; this table was partly designed for this purpose as frustratingly mass renumbering does seem to happen now and then.

The tool doesn't currently store historical FHRS data so the feature you've suggested would require some quite big changes; I'm afraid it's out of scope at the moment, particularly as I have limited time available.

Closing this issue but feel free to continue discussing progress with South Tyneside here if useful.
